### PR TITLE
Add types definition for TS integration

### PR DESCRIPTION
**Pull request by mbalabash**
Hi
It's a great plugin. 
And we should make it accessible for TypeScript developers.

It's related to https://github.com/fastify/fastify-compress/issues/54

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "Fastify compression utils",
   "main": "index.js",
   "dependencies": {
+    "@types/node": "^11.13.9",
```
Comment (evanshortiss) [inline package.json]: @mbalabash can you move this to devDeps. Thanks

Comment (mbalabash) [reply package.json]: Sure

Comment (mbalabash) [reply package.json]: Done

**Comment by Eomm** [comment]
@mbalabash could you add also here the tests for types like you did here https://github.com/fastify/fastify-http-proxy/pull/55?

**Comment by mbalabash** [comment]
Sure. I will add it tomorrow or today night.

**Comment by mbalabash** [comment]
Done @Eomm

File: package.json [hunk 2]
```diff
@@ -19,16 +19,22 @@
...
     "pre-commit": "^1.2.2",
     "standard": "^12.0.0",
-    "tap": "^12.0.0"
+    "tap": "^12.0.0",
+    "typescript": "^3.4.5"
   },
   "scripts": {
-    "unit": "tap test.js",
+    "unit": "tap test/test.js",
     "test": "standard && npm run unit",
```
Comment (Eomm) [inline package.json]:
```suggestion
    "test": "standard && npm run unit && npm run typescript",
```

Comment (mbalabash) [reply package.json]: Fixed

**Comment by mbalabash** [comment]
@Eomm @evanshortiss 
I added fix for merge conflict.

**Pull request merged**